simulariumio/md/md_converter.py

```diff
@@ -62,29 +62,30 @@ def _get_type_name(raw_type_name: str, input_data: MdData) -> float:
         """
         Get the type_name to use for the particle with the given raw type_name
         """
-        if raw_type_name in input_data.display_data:
-            return input_data.display_data[raw_type_name].name
         element_type = guess_atom_element(raw_type_name)
-        if element_type in input_data.display_data:
-            return input_data.display_data[element_type].name + "#" + raw_type_name
+        for name in input_data.display_data:
+            if raw_type_name.lower() == name.lower():
+                return input_data.display_data[name].name
+            if element_type.lower() == name.lower():
+                return input_data.display_data[name].name + "#" + raw_type_name
         return element_type + "#" + raw_type_name
 
     @staticmethod
     def _get_radius(raw_type_name: str, input_data: MdData) -> float:
```

Comment:
not for this PR, but I wonder if this code could be simpler, so we don't have to get display data multiple times for type and radius. It's a bit complicated since these are being called in array functions, but maybe when you're learning more numpy you could think about how to do this better.